stdlib/public/runtime/CMakeLists.txt

```diff
@@ -22,6 +22,10 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "CYGWIN")
   set(swift_runtime_port_sources
       CygwinPort.cpp)
 endif()
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
```

Comment:
Would it make sense to do:

```
if(("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows") OR
   ("${CMAKE_SYSTEM_NAME}" STREQUAL "CYGWIN"))
```

instead of duplicating the logic?